### PR TITLE
Add Academic admin

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes This PR is ready for review. I don't know why the Azure test fails, but all other tests have passed successfully.

File: recipes/academic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 174be3ec5d233c77be7e026052d739b2295ad1e15c07f49b817274c72a7e0050
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
```
Comment (beckermr) [inline recipes/academic/meta.yaml]: This comment can go.

Comment (hoechenberger) [reply recipes/academic/meta.yaml]: Done.

**Comment by scopatz** [comment]
It's ok for azure to fail now

**Comment by beckermr** [comment]
Happy to merge once circle passes!

**Comment by hoechenberger** [comment]
Great, thank you!

--

**Pull request merged**